src/components/ProgressBar/styles.module.scss

```diff
@@ -19,3 +19,7 @@
   display: flex;
   justify-content: space-between;
 }
+
+table {
```

Comment:
In general it's better to give it a class and give the style to that class. Otherwise the next person that adds a table to this file will see a style that doesn't make sense.